File: src/da/mssql.Server/Common/Model/DatabaseServerProperties.cs
namespace mssql.server.Common.Model
{
    public class DatabaseServerProperties
    {
        public IEnumerable<SqlServerProperty> ServerProperties { get; set; }
        public IEnumerable<DatabaseInfo> databaseInfos { get; set; }
        public IEnumerable<RegistryProperty> registryProperties { get; set; }
    }
}

File: src/web/mssql.client/src/app/Model/databaseServerProperties/database-server-properties.ts
import { SqlServerProperty } from './sql-server-property';
import { DatabaseInfo }from './database-info'
import { RegistryProperty } from './registry-property';

export class DatabaseServerProperties {
    serverProperties: SqlServerProperty[];
    databaseInfos: DatabaseInfo[];
    registryProperties: RegistryProperty[];

    constructor(serverProperties: SqlServerProperty[], databaseInfos: DatabaseInfo[], registryProperties: RegistryProperty[]) {
        this.serverProperties = serverProperties;
        this.databaseInfos = databaseInfos;
        this.registryProperties = registryProperties;
    }
}